### PR TITLE
Add command to run ALL the audits

### PR DESCRIPTION
**Pull request by fedorlis**
Adds `all` command, does what it says on the tin.

### CODE REVIEW

File: cmd/image.go [hunk 1]
```diff
@@ -60,9 +60,6 @@ func auditImages(image imgFlags, resource k8sRuntime.Object) (results []Result)
 			break
 		}
 	}
-	for _, result := range results {
```
Comment (klautcomputing) [inline cmd/image.go]: 💯 👍

File: fixtures/audit_all.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: cababilitiesAdded
```
Comment (klautcomputing) [inline fixtures/audit_all.yml]: `cababilitiesAdded` and no caps added 😢

File: README.md [hunk 3]
```diff
@@ -265,6 +266,19 @@ WARN[0000] CPU limit exceeded, it is set to 1 but it must not exceed 500m. Pleas
 WARN[0000] Memory limit exceeded, it is set to 512Mi but it must not exceed 125Mi. Please adjust it!
 ```
 
+<a name="all" />
+
```
Comment (klautcomputing) [inline README.md]: This is a great feature and I think will be the audit that 99% of all the people will run 99% of the time. Thus, it needs to be above all other audits, right next to `autofix`

**Comment by jinankjain** [comment]
@klautcomputing @fedorlis This can also be merged now :)

**Pull request merged**